tgui/packages/tgui/interfaces/MedicalRecords/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const PHYSICALSTATUS2COLOR = {
+  Active: 'good',
+  'Physically Unfit': 'purple',
+  Unconscious: 'average',
+  Deceased: 'bad',
+} as const;
+
+export const PHYSICALSTATUS2DESC = {
+  Active: 'Active. Individual is conscious and healthy.',
+  'Physically Unfit':
+    'Physically Unfit. Individual is conscious, but unhealthy.',
+  Unconscious: 'Unconscious. Individual may require medical attention.',
+  Deceased: 'Deceased. Individual has died and begun to decay.',
+} as const;
+
+export const MENTALSTATUS2COLOR = {
+  Stable: 'good',
+  Watch: 'purple',
+  Unstable: 'average',
+  Insane: 'bad',
+} as const;
```

Comment:
i'd rather `MENTALSTATUS2COLOR` actually output colors, and not a string that will represent a color. "unstable" being "average" is awk to me (especially when coupled with the fact that "Watch" is "Purple")... if you're going to name it such that the constant _will_ output color you should just have it output the color and have the UI do the rest of the inference work.